### DISCUSSION TITLE
Hallo ich habe das problem wenn der trecker eine feld arbeit machen soll steht dann da `Sollte Repariert werden` ich habe schon in den Einstellung geschaut das steht ` Automatische Reparatur` die habe ich aus gemacht aber habe das gleiche problem wieder coursplay ist bei mir auf der version 7.2.0.5

### DISCUSSION THREAD

**Discussion by NewHollandHyper1401** [Courseplay/Courseplay_FS22#2566] [Q&A]
![Screenshot (85)](https://github.com/Courseplay/Courseplay_FS22/assets/134838883/03a781a1-7c3b-4849-9a0c-1af292606635)

**Comment by Legion68** [comment]
das liegt an was anderes aber nicht an CP

Am Di., 30. Mai 2023 um 14:56 Uhr schrieb NewHollandHyper1401 <
***@***.***>:

**Reply by NewHollandHyper1401** [reply]
an was den mods?

**Comment by Legion68** [comment]
mod oder irgend ein script


Am Di., 30. Mai 2023 um 15:23 Uhr schrieb NewHollandHyper1401 <
***@***.***>:

**Reply by NewHollandHyper1401** [reply]
okay danke

**Comment by Tensuko** [comment]
Muss das ganze 2 mal aufgemacht werden?
Du hast es bereits hier:
https://github.com/Courseplay/Courseplay_FS22/discussions/2561

Und die Überschrift ist an sich nicht dafür da seine Leidensgeschichte nieder zu schreiben, dafür ist das große Fenster da.

**Reply by NewHollandHyper1401** [reply]
ja sry
